### PR TITLE
How Kubernetes Add to Your Business

### PR DESCRIPTION
**Pull request by kakorrhaphio**

### CODE REVIEW

**Comment by tylerjohnhaden** [comment]
`How Kubernetes Add to Your Business`
if this is your plan for the title, consider saying
`How Kubernetes Adds to Your Business`
since Kubernetes is a singular noun.

File: posts/how-kubernetes-add-to-your-business.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+There are many benefits of having a Container Orchestration system such as Kubernetes:
+- provisioning and deploying containers
+- redundancy
+- scalability
+- allocation of resources
+- health monitoring
+
+### Conclusion
+Effectively, Kubernetes is powerful tool to use at your helm where your developers have more time to work on the core
```
Comment (tylerjohnhaden) [inline posts/how-kubernetes-add-to-your-business.md]: This says nothing about Ippon, or links the reader to our services.

**Comment by kakorrhaphio** [comment]
> `How Kubernetes Add to Your Business`
> if this is your plan for the title, consider saying
> `How Kubernetes Adds to Your Business`
> since Kubernetes is a singular noun.

Modified.

File: posts/how-kubernetes-add-to-your-business.md (hunk 1 above)
Comment (kakorrhaphio) [reply posts/how-kubernetes-add-to-your-business.md]: I read some of our other blogs, and I'm not sure if we always have to tie things back to ippon and our services. This was intended to just be an informative bloggy blog. I can change plug ippon in if very much desired

Comment (tylerjohnhaden) [reply posts/how-kubernetes-add-to-your-business.md]: My understanding of the audience is that of "non computer people". Correct me if I'm wrong. These people would probably only take away from this article, which brands were mentioned. That's just my take, I'm definitely no expert.

**Comment by tylerjohnhaden** [comment]
I like the image

File: posts/how-kubernetes-add-to-your-business.md (hunk 1 above)
Comment (kakorrhaphio) [reply posts/how-kubernetes-add-to-your-business.md]: hmm alright I'll think of a spin to add to the end, stay tuned 📺

Comment (aalbro) [reply posts/how-kubernetes-add-to-your-business.md]: Tyler is spot on with his suggestions. Consider the business reader that has heard of Kubernetes but doesn't understand why he/she needs it. Then sell them Ippon as the expert that will implement it for them.

Comment (kakorrhaphio) [reply posts/how-kubernetes-add-to-your-business.md]: Well, the main thing is although we're good at a lot of stuff, we aren't exactly kubernetes experts and therefore can't speak to much k8s experience. With that thought process in mind, my goal was to have something for the business readers to have a supplemental blog to point their technical people to.

File: posts/how-kubernetes-add-to-your-business.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+authors:
+- Jeet Gajjar
+tags:
+- kubernetes
+- docker
+- microservices
+
+date: 
+title: "How Kubernetes Adds to Your Business"
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: I'm thinking something more like "How Kubernetes Can Benefit Your Business Needs"

File: posts/how-kubernetes-add-to-your-business.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+image: https://github.com/ippontech/kakorrhaphio/blob/master/images/2019/6/kubernetes-image.png
+---
+
+Monolithic applications have been the industry norm for years, however, we here at Ippon have noticed a shift in the 
+industry wherein many companies in the insurance to e-commerce to banking industries are bound for generating more
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: industry, wherein many (inserted comma)

Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: companies in industries such as insurance, banking, e-commerce, etc. are

File: posts/how-kubernetes-add-to-your-business.md (hunk 2 above)
Comment (kakorrhaphio) [reply posts/how-kubernetes-add-to-your-business.md]: Sure, I like that. Making the change now

File: posts/how-kubernetes-add-to-your-business.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- microservices
+
+date: 
+title: "How Kubernetes Adds to Your Business"
+image: https://github.com/ippontech/kakorrhaphio/blob/master/images/2019/6/kubernetes-image.png
+---
+
+Monolithic applications have been the industry norm for years, however, we here at Ippon have noticed a shift in the 
+industry wherein many companies in the insurance to e-commerce to banking industries are bound for generating more 
+focused and specific applications. These applications exist in the form of [microservices](https://blog.ippon.tech/monolithic-to-microservice-consistency-in-distributed-systems/).  When talking about microservices, its difficult _not_ to talk about containers and container orchestration tools.
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: *it is

Comment (nnightingale14) [reply posts/how-kubernetes-add-to-your-business.md]: *to not talk about

File: posts/how-kubernetes-add-to-your-business.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+When reading into microservices, you and your developers will ask yourselves: `Would you want to spin up a VM,  
+install an OS, find dependencies, manage security, etc or would you like to spin up a container which does all that
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: *etc.  Or would you

File: posts/how-kubernetes-add-to-your-business.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+Monolithic applications have been the industry norm for years, however, we here at Ippon have noticed a shift in the 
+industry wherein many companies in the insurance to e-commerce to banking industries are bound for generating more 
+focused and specific applications. These applications exist in the form of [microservices](https://blog.ippon.tech/monolithic-to-microservice-consistency-in-distributed-systems/).  When talking about microservices, its difficult _not_ to talk about containers and container orchestration tools.
+When reading into microservices, you and your developers will ask yourselves: `Would you want to spin up a VM,  
+install an OS, find dependencies, manage security, etc or would you like to spin up a container which does all that 
+for you in seconds?`
+
+Today, we're going to discuss what is Kubernetes and why your business would benefit from this technology. First
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: *what Kubernetes is and how your business could

File: posts/how-kubernetes-add-to-your-business.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+we'll briefly discuss the differences in microservices and monoliths.
+
+### A Brief History in Microservices
+Before microservices, we had (and still have) monoliths. Applying good development practices, its best to break up
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: *it is

File: posts/how-kubernetes-add-to-your-business.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+Today, we're going to discuss what is Kubernetes and why your business would benefit from this technology. First
+we'll briefly discuss the differences in microservices and monoliths.
+
+### A Brief History in Microservices
+Before microservices, we had (and still have) monoliths. Applying good development practices, its best to break up 
+your main component into modular pieces. These modular pieces, in the form of microservices, are transforming into 
+containers.
+
+Making monoliths modular into microservices have real cost saving benefits both financially and in developer time:
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: *Making monoliths modular into microservices are a dual threat of sorts when it comes to savings -- both in terms of financial costs and also in development time.

File: posts/how-kubernetes-add-to-your-business.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+- Containers help with keeping deployment times low 
+- The main service operates if a component is down or redeploying
+- Adding new features is independent of the entire monolith 
+- Reduces start-up time for developers
+
+To add to the point, this [research](https://www.researchgate.net/publication/316532483_Cost_comparison_of_running_web_applications_in_the_cloud_using_monolithic_microservice_and_AWS_Lambda_architectures) paper from 2015 breaks down the actual cost of using a monolith vs
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: *white paper

File: posts/how-kubernetes-add-to-your-business.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+Making monoliths modular into microservices have real cost saving benefits both financially and in developer time:
+- Containers help with keeping deployment times low 
+- The main service operates if a component is down or redeploying
+- Adding new features is independent of the entire monolith 
+- Reduces start-up time for developers
+
+To add to the point, this [research](https://www.researchgate.net/publication/316532483_Cost_comparison_of_running_web_applications_in_the_cloud_using_monolithic_microservice_and_AWS_Lambda_architectures) paper from 2015 breaks down the actual cost of using a monolith vs 
+microservice architecture using then-current relevant tech stacks. The researchers found that using a microservice 
+architecture in place of monoliths save up to 70% under similar loads.
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: Use either "saves up to" or "can save up to"

File: posts/how-kubernetes-add-to-your-business.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+Now that I've convinced you that microservices have a place in the world, there are some managerial questions to
+address: what happens when there are a bunch of microservices running and how does a development team manage them?
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: *What
*manage them all at once?

File: posts/how-kubernetes-add-to-your-business.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+There are many benefits of having a Container Orchestration system such as Kubernetes:
+- provisioning and deploying containers
+- redundancy
+- scalability
+- allocation of resources
+- health monitoring
+
+### Conclusion
+Effectively, Kubernetes is powerful tool to use at your helm where your developers have more time to work on the core
+ product and increase their velocity. To put it more simply, Kubernetes saves IT/Developers time. Time saved saves
```
Comment (nnightingale14) [inline posts/how-kubernetes-add-to-your-business.md]: *Time savings = cost savings.

File: posts/how-kubernetes-add-to-your-business.md (hunk 9 above)
Comment (kakorrhaphio) [reply posts/how-kubernetes-add-to-your-business.md]: Its a research paper, not a white paper

File: posts/how-kubernetes-add-to-your-business.md (hunk 1 above)
Comment (aalbro) [reply posts/how-kubernetes-add-to-your-business.md]: Think of expert as consulting expert not k8s expert in this context. We don't want to limit sales opportunities based on existing skillsets for hot technologies like k8s. The reason it is a great topic for your blog is the same reason we are looking for these types of projects. We will hire for the right role and we also have resources globally we can tap into. FEED ME K8S PROJECTS :)

**Pull request closed without merging**